Dockerfile

```diff
@@ -1,5 +1,7 @@
-FROM node:10
+FROM node:current-alpine
 
+RUN apk update && apk add bash git
```

Comment:
Use `RUN apk --no-cache add git` to keep the image as slim as possible.
Also, Alpine has built-in shell "Almquist Shell" (`/bin/ash`) which I believe is quite enough for execute hexo-cli.

Reply:
Tks! I will replace it.